<!DOCTYPE html>
<html lang="en-US">
<head>
    <!--
  <link rel="preload" href="./lib/font-awesome/webfonts/fa-brands-400.woff2" as="font" type="font/woff2" crossorigin="anonymous">
  <link rel="preload" href="./lib/font-awesome/webfonts/fa-regular-400.woff2"
  as="font" type="font/woff2" crossorigin="anonymous"> -->
  <link rel="preload" href="./lib/font-awesome/webfonts/fa-solid-900.woff2" as="font" type="font/woff2" crossorigin="anonymous">

<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">


  <script type="text/javascript" src="./js/cactus.js"></script>
  <link rel="stylesheet" href="./css/cactus.css" type="text/css">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>RD2Bool Research Project</title>
  <link rel = 'canonical' href = './'>
  
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="robots" content="all,follow">
  <meta name="googlebot" content="index,follow,snippet,archive">
  <link rel="stylesheet" href="./css/styles.7ed8b1ae3186b58773b8434f18eca93949a2259bb6155e72056d11ac6ecac5daa512309dda1f9260d4d596b8fd492e7561390bbe935dae92eff69a14e9395b13.css">

    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

    <style type="text/css">
* [lang=fr] {
    font-style: italic;
}

h2 {
    color: #355ae3;
}
#title h1 {
    color: #2845ae;
}

    </style>

</head>

<body class="max-width mx-auto px3 ltr">
  <div class="content index py4">

    <header id="header">
  <a href="./">
  
  
  <div id="title">
    <h1>RD2Bool Research Project</h1>
  </div>
  </a>
  <div id="nav">
    <ul>
      <li class="icon">
        <a href="#" aria-label="Menu"><i class="fas fa-bars fa-2x" aria-hidden="true"></i></a>
      </li>
        <li><a href="#members">Members</a></li>
        <!--
        <li><a href="#publications">Publications</a></li>
        <li><a href="#code">Code</a></li>
        -->
    </ul>
  </div>
</header>

  <section id="about">
  
  <p><img src="./img/Logo-ANR-bleu-RVB.png#sponsor" alt=""></p>
<h2>Reconciling data driven and prior knowledge approaches to infer cell-type specific ensemble Boolean models using single cell multi-omics time courses</h2>
<p>Research project funded by <a href="http://www.agence-nationale-recherche.fr">ANR</a> (French National Research Agency; grant <a href="https://anr.fr/Project-ANR-23-CE45-0014">ANR-23-CE45-0014</a>).</p>

<p>An explosion in the availability of transcriptomics data, especially in single cells, has produced a surge in methods to infer regulatory networks directly from data, but a systematic comparison with prior knowledge-based models is still lacking. A major question is whether we should consider that specific cell types require specific networks and models.<br>RD2Bool will test the appropriateness of different inference and modelling frameworks against state-of-the-art single cell datasets, exploiting our highly complementary expertise on in vitro cultures, immune cell multi-omics and bioinformatics, simulations of literature-based Boolean models and frameworks for Boolean model reconstruction.<br>Our specific aims are to: 1) Produce a state-of-the-art time course of single cell multi-omics; 2) Compare data-driven to prior knowledge-based methods for inference of regulatory networks; 3) Compare and combine different strategies for Boolean model inference; 4) Model populations of specific interacting cell types.<br>The main outcome will be an assessment of performance of data-driven model building approaches for the construction of cell type specific models of the major immune cell types and the prediction of their roles in diverse immune diseases.<br>The simultaneous measurements of cell surface markers as well as RNAseq, determining cell types (immunologists based definition) vs. cell states, will be a unique opportunity to test regulatory networks and model inference methods. We will provide an optimised set of tools to perform data-driven model inference and in silico simulations for exploration of cell-type-specific models. The models for the main immune cell types in blood, with a focus on myeloid cells’ differentiation, will be relevant for immune alteration associated with ageing, covid, and cancer, but the tools developed will be completely general and could be applied to any multi-cellular system of interest and potentially to multi-level systems beyond biology.</p>
<p><em>Principal investigator: <a href="https://www.crct-inserm.fr/en/netbio2_en/">Vera Pancaldi</a> (INSERM, CRCT, Toulouse, France)</em></p>


<!--
  <section id="publications">
    <h1>Publications</h1>
    <p><a href="https://hal-anr.archives-ouvertes.fr/search/index/?q=%2A&anrProjectReference_s=ANR-23-CE45-001&sort=producedDate_tdate+desc"
        >List of publications</a></p>
  </section>
-->

<section id="members">
    <h1>Members</h1>

    <h2>CRCT, Toulouse</h2>

    <ul>
        <li>
            <a href="https://www.crct-inserm.fr/en/netbio2_en">Vera Pancaldi</a>, Principal investigator
        </li>
        <li>
            Malivna Marku, Post-doc
        </li>
        <li>
            Hugo Chenel, PhD student
        </li>
    </ul>

    <h2>Institut Curie, Paris</h2>
    <ul>
        <li>
            <a href="https://curie.fr/personne/laurence-calzone">Laurence Calzone</a>, Site coordinator
        </li>
    </ul>

    <h2>LaBRI, Bordeaux</h2>
    <ul>
        <li>
            <a href="https://loicpauleve.name">Loïc Paulevé</a>, Site coordinator
        </li>
    </ul>



  </div>
</body>

<link rel="stylesheet" href=./lib/font-awesome/css/all.min.css>
<script src=./lib/jquery/jquery.min.js></script>
<script src=./js/main.js></script>
</html>
